Networking/Networking/Model/StripeAccount.swift

```diff
@@ -0,0 +1,128 @@
+        self.hasOverdueRequirements = hasOverdueRequirements
+        self.currentDeadline = currentDeadline
+        self.statementDescriptor = statementDescriptor
+        self.defaultCurrency = defaultCurrency
+        self.supportedCurrencies = supportedCurrencies
+        self.country = country
+        self.isCardPresentEligible = isCardPresentEligible
+    }
+
+    /// The public initializer for WCPay Account.
+    ///
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        let status = try container.decode(WCPayAccountStatusEnum.self, forKey: .status)
+        let isLiveAccount = try container.decode(Bool.self, forKey: .isLive)
+        let isInTestMode = try container.decode(Bool.self, forKey: .testMode)
+        let hasPendingRequirements = try container.decode(Bool.self, forKey: .hasPendingRequirements)
+        let hasOverdueRequirements = try container.decode(Bool.self, forKey: .hasOverdueRequirements)
+        let currentDeadline = try container.decodeIfPresent(Date.self, forKey: .currentDeadline)
+        let statementDescriptor = try container.decode(String.self, forKey: .statementDescriptor)
+        let currencyContainer = try container.nestedContainer(keyedBy: CurrencyCodingKeys.self, forKey: .storeCurrencies)
+        let defaultCurrency = try currencyContainer.decode(String.self, forKey: .defaultCurrency)
+        let supportedCurrencies = try currencyContainer.decode([String].self, forKey: .supported)
+        let country = try container.decode(String.self, forKey: .country)
+        /// Ignore the state of the In-Person Payments Beta flag (`.cardPresentEligible`). Allow any site. See #5030
+        /// let cardPresentEligible = try container.decodeIfPresent(Bool.self, forKey: .cardPresentEligible) ?? false
+
+        self.init(
+            status: status,
+            isLiveAccount: isLiveAccount,
+            isInTestMode: isInTestMode,
+            hasPendingRequirements: hasPendingRequirements,
+            hasOverdueRequirements: hasOverdueRequirements,
+            currentDeadline: currentDeadline,
+            statementDescriptor: statementDescriptor,
+            defaultCurrency: defaultCurrency,
+            supportedCurrencies: supportedCurrencies,
+            country: country,
+            isCardPresentEligible: true
+        )
+    }
+}
+
+public extension StripeAccount {
+    static let noAccount = StripeAccount(
+        status: .noAccount,
+        isLiveAccount: false,
+        isInTestMode: false,
+        hasPendingRequirements: false,
+        hasOverdueRequirements: false,
+        currentDeadline: nil,
+        statementDescriptor: "",
+        defaultCurrency: "",
+        supportedCurrencies: [],
+        country: "",
+        isCardPresentEligible: false
+    )
+}
+
+private extension StripeAccount {
+    enum CodingKeys: String, CodingKey {
+        case status = "status"
```

Comment:
nit: maybe we can leave the value since it's the same as the key (like `country` case below) 😄 
```suggestion
        case status
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5799/commits/0ce7cecbf3ae80f1ce41cd06bfa2f3be2686063c in both this model and the model this was based on